hawkmoth/util/compiler.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021 Jani Nikula <jani@nikula.org>
+# Licensed under the terms of BSD 2-Clause, see LICENSE for details.
+"""
+Compiler helpers
+================
+
+This module provides helper functions to access compiler information outside of
+the Clang Python Bindings, for example system include paths.
+"""
+
+import subprocess
+
+def _removesuffix(s, suffix):
+    if suffix and s.endswith(suffix):
+        return s[:-len(suffix)]
+    else:
+        return s[:]
+
+def _get_paths_from_output(output):
+    started = False
+    for line in output.splitlines():
+        if not started:
+            if line == '#include <...> search starts here:':
+                started = True
+            continue
+
+        if line == 'End of search list.':
+            break
+
+        line = _removesuffix(line, '(framework directory)')
```

Comment:
This is for Macs if I remember correctly. Maybe we should place a small comment on it so we're not tempted to remove it without 1st confirming there no longer is a need for it.

Reply:
Ack.